cpp/ql/src/experimental/Security/CWE/CWE-758/DangerousUseMallocWithZeroSize.ql

```diff
@@ -0,0 +1,164 @@
+/**
+ * @name Dangerous use malloc with zero size.
+ * @description The behavior of the malloc function is not defined when the size value is zero.
+ * @kind problem
+ * @id cpp/dangerous-use-malloc-with-zero-size
+ * @problem.severity warning
+ * @precision medium
+ * @tags correctness
+ *       security
+ *       external/cwe/cwe-758
+ */
+
+import cpp
+import semmle.code.cpp.commons.Exclusions
+import semmle.code.cpp.valuenumbering.GlobalValueNumbering
+
+/** Holds if there are checks above the call */
```

Comment:
Can you extend the QLDoc to explain _which_ checks this predicate is looking for?

Reply:
i fixed it. Have a look, please